src/queue/consumers/oneidentity/utils/isProposalMessageData.ts

```diff
@@ -0,0 +1,12 @@
+import { ProposalMessageData } from '../../../../models/ProposalMessage';
+
+// For OneIdentity integration, we need these fields to be present in the message
+export function isProposalMessageData(
```

Comment:
if this is really needed as a separate function I would rather call it `validateBasicProposalMessageData` than `isProposalMessageData`. Just to keep the same practice as the other validation files.

Reply:
I replaced it with the available proposal validator function.